### PR TITLE
NTP-788: Update Cookie Name Details

### PR DESCRIPTION
**Pull request by Kolapo-Abolarinwa**
## Context
On the cookie page:
The correct information about all the cookies used should be displayed.

## Changes proposed in this pull request

- Change the Cookie name from ‘.FindATuitionPartner.search-page.shortlistedTps' to  '.FindATuitionPartner.Shortlist’
- Update provided details for ‘.FindATuitionPartner.Mvc.CookieTempDataProvider’ cookie, to reflect what it’s currently used for.

## Guidance to review
On the cookie page:
- The summary of the two named cookies should provide/display the correct detail of what they are used for.
- The name change for ‘.FindATuitionPartner.Mvc.CookieTempDataProvider’ should be reflected.

## Link to Jira ticket

[NTP-788](https://dfedigital.atlassian.net/jira/software/projects/NTP/boards/135/backlog?selectedIssue=NTP-788&text=788)

## Things to check

- [X] Title is in the format "NTP-XXX: Short description" where NTP-XXX is the Jira ticket reference
- [ ] Code and tests follow the [coding standards](/docs/coding-standards.md)
- [X] `dotnet format` has been run in the repository root
- [ ] Test coverage of new code is at least 80%
- [ ] All UI related acceptance criteria specified in the ticket have been added to the relevant [cypress test feature file(s)](/UI/cypress/e2e/)
- [ ] Unless explicitly mentioned in the ticket, all UI work should have been tested as working on desktop and mobile resolutions with JavaScript on and off
- [ ] Database migrations can be applied to the current codebase in main without causing exceptions
- [ ] Debug logging has been added after all logic decision points
- [ ] All [automated testing](/README.md#testing) including accessibility and security has been run against your code
- [ ] **PR deployment has been signed off by wider team**

### CODE REVIEW

File: UI/Pages/Cookies.cshtml [hunk 1]
```diff
@@ -88,75 +89,81 @@
...
+            </tr>
+            <tr>
+                <td class="govuk-table__cell">.FindATuitionPartner.Mvc.CookieTempDataProvider</td>
+                <td class="govuk-table__cell">Stores the return url on the cookie page</td>
+                <td class="govuk-table__cell">When the browser is closed</td>
+            </tr>
+            <tr>
+                <td class="govuk-table__cell">.FindATuitionPartner.Shortlist</td>
+                <td class="govuk-table__cell">Stores your shortlisted tuition partners</td>
+                <td class="govuk-table__cell">24 hours from the last tuition partner added to the shortlist</td>
```
Comment (ashward) [inline UI/Pages/Cookies.cshtml:164]: I'd just simplify this and say 24 hours.
```suggestion
                <td class="govuk-table__cell">24 hours</td>
```

File: UI/Pages/Cookies.cshtml [hunk 2]
```diff
@@ -66,7 +67,7 @@
         </ul>
 
         <form method="post">
-            <input asp-for="ReturnUrl" value="@Model.ReturnUrl" hidden=true />
+            <input asp-for="ReturnUrl" value="@Model.ReturnUrl" hidden=true/>
```
Comment (ashward) [inline UI/Pages/Cookies.cshtml:70]: The updated formatting seems a little strange in this file. What formatter are you using?

Comment (Kolapo-Abolarinwa) [reply UI/Pages/Cookies.cshtml:70]: It's the Rider equivalent to Visual studio's Control + k, Control +d

**Review by ashward — APPROVED** [review]
lgtm

**Pull request merged**